cdap-system-app-api/src/main/java/io/cdap/cdap/api/service/AbstractSystemService.java

```diff
@@ -24,7 +24,7 @@
  * system namespace.
  */
 @Beta
-public abstract class AbstractSystemService extends AbstractService<SystemServiceConfigurer> {
+public abstract class AbstractSystemService extends AbstractService<SystemServiceConfigurer, SystemServiceContext> {
```

Comment:
I believe this is a backwards compatible change right? Existing code doesn't break?

Reply:
Yes, the wrangler, delta, and studio all can build without problem.